staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher_testing_utils_test.go

```diff
@@ -93,9 +95,6 @@ func compactStorage(c *Cacher) storagetesting.Compaction {
 		if c.watchCache.resourceVersion < rv {
 			t.Fatalf("Can't compact into a future version: %v", resourceVersion)
 		}
-		if rv < c.watchCache.listResourceVersion {
```

Comment:
Don't understand why this check is here.

Reply:
It was more of a safety protection - I wanted to ensure we're really compacting something.
But yeah - it's technically not needed.

Reply:
Thanks for confirming.